querybook/webapp/ui/Modal/Modal.scss

```diff
@@ -58,6 +58,7 @@
             .Modal-content {
                 max-height: inherit;
                 height: auto;
+                overflow-y: hidden;
```

Comment:
this doesn't do anything? since line 62 removes it

Reply:
🤦 i meant overflow-x